src/perform_step/extrapolation_perform_step.jl

```diff
@@ -48,17 +48,22 @@ function initialize!(integrator,cache::RichardsonEulerConstantCache)
   integrator.fsallast = zero(integrator.fsalfirst)
   integrator.k[1] = integrator.fsalfirst
   integrator.k[2] = integrator.fsallast
+
+  cache.order = 3
 end
 
 function perform_step!(integrator,cache::RichardsonEulerConstantCache,repeat_step=false)
   @unpack t,dt,uprev,f,p = integrator
-  @unpack m = cache
+  @unpack dtpropose, T, order, order_max, last_work, work, A = cache
   @muladd u = @. uprev + dt*integrator.fsalfirst
-  T = zeros(typeof(uprev), (m,m))
+  A = 1 + 1
+
   T[1,1] = u
+  dtpropose = dt #doubt but error_1 isn't defined
+  work = A/dtpropose
 
   halfdt = dt/2
-  for i in 2:m
+  for i in 2:order_max
```

Comment:
order?

Reply:
size of `T` is `(max_order, max_order)`, hence I had written something like this in my pseudo code. Now I had changed this with `size(T)[0]`. Is this fine?